### PR TITLE
Fix/tbd 32/fix unittest

### PR DESCRIPTION
**Pull request by SergiiTao**
Ticket: https://oat-sa.atlassian.net/browse/TBD-32
Unit test fixed

### CODE REVIEW

**Comment by Yury-Razhkou** [comment]
There is also 2 folders `Unit` and `unit`. Maybe we can merge it in this PR?

File: test/Unit/models/Mapper/ResultMapperTest.php [hunk 1]
```diff
@@ -19,7 +19,7 @@
  *
  */
 
-namespace oat\taoResultServer\test\unit\models\Mapper;
+namespace oat\taoResultServer\test\Unit\models\nit\models\Mapper;
```
Comment (wazelin) [inline test/Unit/models/Mapper/ResultMapperTest.php:22]: Something really weird happened to this namespace.
Moreover, in order to truly follow PSR-4, it follow the same case as the file path, so it should be like that 👇.
```suggestion
namespace oat\taoResultServer\test\unit\models\Mapper;
```

Same applies to all the other namespaces.

Comment (SergiiTao) [reply test/Unit/models/Mapper/ResultMapperTest.php]: PHPStorm autocompletion happened :)
fixed. thanks

Comment (SergiiTao) [reply test/Unit/models/Mapper/ResultMapperTest.php]: if the case should be the same as in the path `Unit` should be uppercased.

Comment (wazelin) [reply test/Unit/models/Mapper/ResultMapperTest.php]: Huh, interesting.
Looks like at some point it was in lower case, and those sort of changes (changing a letter case of a directory) don't make into a diff, unless all the underlying files are modified. So I still have it in lower case locally.

**Pull request merged**